app/models/user.rb

```diff
@@ -16,6 +16,7 @@ class User < ApplicationRecord
   has_many   :miq_widget_contents, :dependent => :destroy
   has_many   :miq_widget_sets, :as => :owner, :dependent => :destroy
   has_many   :miq_reports, :dependent => :nullify
+  has_many   :service_orders, :dependent => :nullify
```

Comment:
Wouldn't we always get the list of service_orders via a user object?

Maybe, we'd want to see order history or something for the tenant/group, and not just user?  Is that what you're thinking?